README.md

```diff
@@ -1,13 +1,29 @@
+# YaST - rewrite of the storage library
 
-This is a technoligy preview to demonstrate:
+[![Build Status](https://travis-ci.org/yast/yast-storage-ng.svg?branch=master)](https://travis-ci.org/yast/yast-storage-ng)
+[![Coverage
+Status](https://coveralls.io/repos/github/yast/yast-storage-ng/badge.svg?branch=master)](https://coveralls.io/github/yast/yast-storage-ng?branch=master)
+[![Code
+Climate](https://codeclimate.com/github/yast/yast-storage-ng/badges/gpa.svg)](https://codeclimate.com/github/yast/yast-storage-ng)
 
-- libstorage-ng supports:
+yast2-storage-ng is a reimplementation of the YaST storage library (yast2-storage)
+based on the also reimplemented library for storage manager (libstorage-ng).
 
-  * filesystems on disks without a partition table
+This module includes some working but very limited prototypes capable of actions
+such as partitioning or proposing a filesystem layout for installation.
 
-  * MD RAIDs can be changed to be partitioned or unpartitions on the fly
+The module is **entirely unsupported**.
 
-- use of libstorage-ng from Ruby/YaST without legacy code
+## Installation
 
-The module is entirely unsupported.
+If you are brave enough to install this module in a (open)SUSE system, despite
+the facts that is not supported and can produce data loss, you can perform the
```

Comment:
Ok, but then I will remove the bold "entirely unsupported" from the previous paragraph. Let's keep the calm. :wink: